corehq/motech/dhis2/views.py

```diff
@@ -364,7 +364,12 @@ def get_update_form(self, instance=None):
 def send_dataset_now(request, domain, pk):
     dataset_map = SQLDataSetMap.objects.get(domain=domain, pk=pk)
     send_date = datetime.utcnow().date()
-    result = send_dataset(dataset_map, send_date)
+
+    result = send_dataset(
+        dataset_map,
+        send_date,
+        log_resource_url=reverse('motech_log_list_view', args=[domain])
```

Comment:
is there a reason you need to pass this in, instead of generating it inside the method? You would then be able to use this helper, which encodes the params: https://github.com/dimagi/commcare-hq/blob/master/corehq/util/view_utils.py#L116-L126

Reply:
@proteusvacuum Good question. I initially wanted to, because having to suddenly also pass in a `log_resource_url` doesn't really make sense if all I want to do is `send_dataset`. I need the domain for the `reverse` (but passing `domain` makes even less sense) so I stuck with passing the `url`..._but_...I just realized I can just get the domain from the `dataset_map`.